security/aws/rds_unencrypted/AWS_Unencrypted_RDS_Instances.pt

```diff
@@ -0,0 +1,493 @@
+        if((param_exclude_tags_lower.indexOf((tag['tagKey']).toLowerCase()) !== -1) || (param_exclude_tags_lower.indexOf((tag['tagKey']+'='+tag['tagValue']).toLowerCase()) !== -1)){
+          isTagMatched = true;
+        }
+        // Constructing tags with comma separated to display in detail_template
+        if((tag['tagValue']).length > 0){
+          tagKeyValue = tagKeyValue + ', '+ tag['tagKey']+'='+tag['tagValue']
+        }else{
+          tagKeyValue = tagKeyValue + ', '+ tag['tagKey']
+        }
+      }
+
+      var publicly_Accessible = rds['publicly_Accessible'];
+      if(publicly_Accessible === true){
+        publicly_Accessible = 'Yes';
+      }else{
+        publicly_Accessible = 'No';
+      }
+
+      var delete_Protection = rds['delete_Protection'];
+      if(delete_Protection === true){
+        delete_Protection = 'Yes';
+      }else{
+        delete_Protection = 'No';
+      }
+
+      //If the RDS instance tag does not match with entered param_exclude_tags, then check if RDS instance is Unencrypted.
+      if(!(isTagMatched) && (rds['storage_Encrypted'] === false)){
+        content.push({
+          dbInstance_Identifier: rds['dbInstance_Identifier'],
+          region: rds['region'],
+          availability_zone: rds['availability_zone'],
+          sec_availability_zone: rds['sec_availability_zone'],
+          publicly_Accessible: publicly_Accessible,
+          dbName: rds['dbName'],
+          db_instance_status: rds['db_instance_status'],
+          delete_Protection: delete_Protection,
+          tagKeyValue:(tagKeyValue.slice(2)),
+          dbInstanceArn: rds['dbInstanceArn'],
+          dbCluster_Identifier: rds['dbCluster_Identifier']
+        })
+      }
+    }
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_rds_instance_list" do
+  validate $ds_rds_unencrypted_instance_list do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} unencrypted RDS Instances Found in AWS"
+    detail_template <<-EOS
+# List of unencrypted RDS Instances
+| RDS Instance Name | Region | Availability Zone | Sec Availability Zone | Publicly Accessible | DB Name | DB Instance Status | Delete Protection Enabled | TAGs |
+| ----------------- | ------ | ----------------- | --------------------- | ------------------- | ------- | ------------------ | ------------------------- | ---- |
+{{ range data -}}
+| {{.dbInstance_Identifier}} | {{ .region }} | {{ .availability_zone }} |  {{ .sec_availability_zone }} |{{ .publicly_Accessible }} | {{ .dbName }} | {{ .db_instance_status }} | {{.delete_Protection}} | {{.tagKeyValue}} |
+{{ end -}}
+###### Note: Kindly refer 'Enable Delete Action' section in the [README](https://github.com/rightscale/policy_templates/tree/master/security/aws/rds_unencrypted) file to perform delete operation.
```

Comment:
Note: Kindly refer -> Note: Please refer to

Reply:
I have made the change.